wiki/Team-Scrum-process.md

```diff
@@ -1,42 +1,42 @@
 ## Our current Scrum process is the following:
 
-### General 
+### General
 - One sprint is usually three weeks long.
-- The teams core work time is monday (9am-3pm) and wednesday (9am-3pm)
+- The teams' core work time is Monday (9 am-3 pm) and Wednesday (9 am-3 pm)
 - Each member of the team works about 16 hours per week
 
-Update: Starting March 8, the team will work on the project four days a week. Monday-Wednesday will be the core work days that each team member will work. The fourth workday can be any of the days from Thursday through Sunday. 
+Update: Starting March 8, the team will work on the project four days a week. Monday-Wednesday will be the core workdays that each team member will work. The fourth workday can be any of the days from Thursday through Sunday.
```

Comment:
We can update this now as we are starting with full time work next monday

Reply:
Same goes for the daily scrum

Reply:
+1